src/screens/draft/index.tsx

```diff
@@ -45,95 +54,93 @@ const Draft = ({route}: TNavigation<Routes.DRAFT>) => {
     reValidateMode: 'onSubmit',
   });
 
-  const averageSkillTeamA = Math.floor(
-    teamA.reduce(
-      (accumulator: number, initialValue: IPlayer) =>
-        accumulator + initialValue.skill,
-      0,
-    ) / teamA.length,
-  );
-
-  const averageSkillTeamB = Math.floor(
-    teamB.reduce(
-      (accumulator: number, initialValue: IPlayer) =>
-        accumulator + initialValue.skill,
-      0,
-    ) / teamB.length,
-  );
-
   const listSeparator = () => <View style={styles.separator} />;
 
   const renderItemTeamA = (data: ListRenderItemInfo<IPlayer>) => {
     return (
-      <Text>
-        {data.item.firstName} {data.item.lastName}
-      </Text>
+      <View style={{width: '100%'}}>
+        <Text style={styles.teamARow}>
+          {data.item.firstName} {data.item.lastName}
+        </Text>
+      </View>
     );
   };
 
   const renderItemTeamB = (data: ListRenderItemInfo<IPlayer>) => {
     return (
-      <Text>
-        {data.item.firstName} {data.item.lastName}
-      </Text>
+      <View style={{width: '100%'}}>
+        <Text style={styles.teamBRow}>
+          {data.item.firstName} {data.item.lastName}
+        </Text>
+      </View>
     );
   };
 
   const onConfirmDate = (value: Date) => {
     setDate(value);
     modalDatePicker(openDatePicker);
-    setValue('date', value);
+    setValue('date', value.toLocaleDateString());
   };
 
   const onConfirmTime = (value: Date) => {
     setTime(value);
     modalTimePicker(openDatePicker);
-    setValue('time', value);
+    setValue(
+      'time',
+      value.toLocaleTimeString('en-US', {
+        hour12: false,
+        hour: '2-digit',
+        minute: '2-digit',
+      }),
+    );
   };
 
-  const onSubmit = (data: any) => {
+  const onSubmit = (data: FormData) => {
     const dataMatch = {
+      id: match?._id || '',
       field: data.field,
-      date: format(data.date, 'd/M/yyyy'),
-      time: data.time,
+      date: format(date, 'MM-dd-yyyy'),
+      time: format(time, 'hh:mm'),
     };
-    Alert.alert(
-      'Match',
-      dataMatch.field +
-        '\n' +
-        dataMatch.date +
-        '\n' +
-        format(dataMatch.time, 'h:mm a'),
-      [{text: 'OK'}],
-    );
+
+    dispatch(updateMatch(dataMatch));
+    navigation.navigate(Routes.HOME);
```

Comment:
We are not handling potential errors from the dispatch. We should handle this case with a customToast (the component and use case is already done on the profile screen). If we encounter and error, the navigation to the Home Screen should not be done